fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchReferenceTest.java

```diff
@@ -132,15 +131,7 @@ public void testSearchWithRelativePatientIdUsingId() {
         assertResponse(response, Response.Status.OK.getStatusCode());
         Bundle bundle = response.readEntity(Bundle.class);
         assertNotNull(bundle);
-        assertTrue(bundle.getEntry().size() >= 1);
-        Patient p = null;
-        for (Bundle.Entry entry : bundle.getEntry()) {
-            if (entry.getResource() != null && entry.getResource() instanceof Patient) {
-                p = (Patient) entry.getResource();
-            }
-        }
-        assertNotNull(p);
-        assertEquals("3001", p.getManagingOrganization().getReference().getValue());
+        assertTrue(bundle.getEntry().size() == 0);
```

Comment:
should we have a tenant 1 test where this test passes and works? as well as this modified one?

Reply:
> > Updated a few spec examples, since a few included invalid resource types in references that did not match the spec.
> 
> We usually try to keep some kind of change log when we edit spec artifacts.
> The other option is to just mark them as invalid in our index...then they actually just become negative test cases.

I think all the spec example references that didn't include valid resource types started with "http:" or ":https", so I can revert those spec example changes if I skip validation of those.

Reply:
I'll update this to use tenant1 and set tenant1 to have checkReferenceResourceType to false, so these cases can be exercised.

Reply:
Oh, actually, the config parameter is not tenant-specific. If I could change that, then I could add this test.

Reply:
Since we don't have a requirement to change `fhirServer/core/checkReferenceTypes` into a per-tenant config parameter, I'll leave this test as is. There are tests in AbstractIncludeRevincludeTest.java (fhir-persistence) and AbstractReverseChainTest.java (fhir-persistence) that exercise searches when a logical-ID only reference is used. So it is at least exercised somewhere.